src/ctia/lib/riemann.clj

```diff
@@ -0,0 +1,124 @@
+(ns ctia.lib.riemann
+  (:require [clojure.string :as str]
+            [clojure.tools.logging :as log]
+            [riemann.client :as riemann]
+            [ctia.properties :as prop]
+            [ctia.lib.utils :as utils])
+  (:import [clojure.lang ExceptionInfo]))
+
+(defn request->event
+  [request extra-fields]
+  (into {:uri (str (:uri request))
+         :_params (utils/safe-pprint (:params request))
```

Comment:
Why this param is prefixed with `_`? I don't remember :(

Reply:
This is because kibana will not index field starting with `_`.